packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/ButtonDemo/ButtonDemo.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button, ButtonProps } from '@patternfly/react-core';
+
+export class ButtonDemo extends React.Component {
+  myButtonProps: ButtonProps = {
+    component: 'button',
+    href: 'https://pf-next.com/',
```

Comment:
Looks like you have an href here ... not sure if you can verify this with cypress easily.  If not maybe change these to show a label you can verify.